viam_cartographer.go

```diff
@@ -355,9 +367,15 @@ func parseCartoAlgoConfig(configParams map[string]string, logger logging.Logger)
 			}
 			cartoAlgoCfg.MinRange = float32(fVal)
 		case "min_range_meters":
-			fVal, err := strconv.ParseFloat(val, 32)
-			if err != nil {
-				return cartoAlgoCfg, err
+			var fVal float64
+			var err error
+			if val == "" {
+				fVal = float64(defaultCartoAlgoCfg.MinRange)
+			} else {
+				fVal, err = strconv.ParseFloat(val, 32)
+				if err != nil {
+					return cartoAlgoCfg, err
```

Comment:
You've copied and pasted the same fix twice. Would it be worth making this a helper function? That would make it clearer that it's the same code in both places, and if you have future bugs to fix or future features to add, they only need to be changed in one location (no chance of forgetting about the second one). 

but I don't feel strongly: leave as-is if you think that's the right choice.

Reply:
Something like:
```go
func parseOrDefault(val string, default float64) (float64, error) {
    if val == "" {
        return default, nil
    }
    return strconv.ParseFloat(val, 32)
}
```
and then you can pass in the MaxRange or the MinRange as the default in the two places it would be used. Admittedly, it's not as long a helper function as I expected, so maybe not worth the hassle.

Reply:
making it a helper function lets me put it in every other relevant case, so def a good callout